osquery/extensions/impl_thrift.cpp

```diff
@@ -318,7 +318,16 @@ void ExtensionRunnerInterface::serve() {
 }
 
 void ExtensionRunnerInterface::connect() {
+#ifdef WIN32
+  // Specify a security descriptor string, in SDDL format, that allows access
+  // (to the named pipe) only to the System and the Built-in Administrators.
+  const std::string& securityDescriptor = "D:(A;;FA;;;SY)(A;;FA;;;BA)";
```

Comment:
Is it possible to inspect who is running osquery(d|i) and also give them access?

Reply:
would be nice to have this feature (give access to user who started osqueryd/i) for testing. Tests account might not be Admin.

Reply:
Yes, this should be possible, and I'm adding that now.